internal/eval/store_eval.go

```diff
@@ -5000,6 +5000,54 @@ func evalLINSERT(args []string, store *dstore.Store) *EvalResponse {
 	return makeEvalResult(res)
 }
 
+func evalLINDEX(args []string, store *dstore.Store) *EvalResponse {
+	if len(args) != 2 {
+		return makeEvalError(diceerrors.ErrWrongArgumentCount("LINDEX"))
+	}
+
+	key := args[0]
+	index, err := strconv.ParseInt(args[1], 10, 64)
+
+	if err != nil {
+		return makeEvalError(errors.New("-ERR value is not an integer or out of range"))
```

Comment:
Shall we reuse this var here? `IntOrOutOfRangeErr`
We can create similar in `migrated_errors.go` and reuse it.